tests/performance/specs/admin-l10n.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { test } from '@wordpress/e2e-test-utils-playwright';
+
+/**
+ * Internal dependencies
+ */
+import { camelCaseDashes } from '../utils';
+
+const results = {
+	timeToFirstByte: [],
+	largestContentfulPaint: [],
+	lcpMinusTtfb: [],
+};
+
+test.describe( 'Admin (L10N)', () => {
+	test.use( {
+		storageState: {}, // User will be logged out.
+	} );
+
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+		await requestUtils.updateSiteSettings( {
+			language: 'de_DE',
+		} );
+	} );
+
+	test.afterAll( async ( { requestUtils }, testInfo ) => {
+		await testInfo.attach( 'results', {
+			body: JSON.stringify( results, null, 2 ),
+			contentType: 'application/json',
+		} );
+		await requestUtils.updateSiteSettings( {
+			language: '',
+		} );
+	} );
+
+	const iterations = Number( process.env.TEST_RUNS );
+	for ( let i = 1; i <= iterations; i++ ) {
+		test( `Measure load time metrics (${ i } of ${ iterations })`, async ( {
+			page,
+			metrics,
+		} ) => {
+			await page.goto( '/' );
```

Comment:
Is this right? I would expect this to be something like `/wp-admin/index.php`. Or where else do you ensure it's actually visiting the admin URL?